splinepy/helpme/reparameterize.py

```diff
@@ -85,3 +85,67 @@ def parametric_axes(spline, permutation_list, inplace=True):
     else:
         _utils.log.debug("  returning permuted spline")
         return type(spline)(**dict_spline)
+
+
+def invert_axes(spline, axes, inplace=False):
+    """
+    Flip the parametric axes
+
+    This function flips parametric axes without changing the parametric bounds
+    of the spline. This can be used, e.g., to achieve conformaty on a
+    multipatch interface
+
+    Parameters
+    ----------
+    spline: Spline
+    axes : list
+        List of axes to be flipped
+    inplace: bool
+        Default is True. If True, modifies spline inplace, else, returns
+        a modified_spline.
+
+    Returns
+    -------
+    modified_spline : type(spline)
+        spline with reordered parametric dimensions. iff `inplace=True`.
+    """
+    from splinepy.helpme.multi_index import MultiIndex as _MultiIndex
```

Comment:
I know this is done at that clean up before, but we don't have to import with underscore within a function. This is not a change request

Reply:
I'll keep a note for the future, thanks!